packages/block-library/src/query/index.js

```diff
@@ -51,6 +51,16 @@ export const settings = {
 					},
 					{
 						name: 'core/post-date',
+						attributes: {
```

Comment:
The example right now is not working and I think that might probably be a general block bindings things, which I'll investigate now.

Nonetheless, this change should be here.

Reply:
We might return early for some store subscriptions when `isPreviewMode` is true. This is usually done for performance.

Here's one related PR I remember - #60544.